modules/userdirectory-ldap/src/main/java/org/opencastproject/userdirectory/ldap/OpencastLdapAuthoritiesPopulator.java

```diff
@@ -354,8 +371,11 @@ private void addAuthorities(Set<GrantedAuthority> authorities, final String[] va
     if (values != null) {
       Organization org = securityService.getOrganization();
       if (!organization.equals(org)) {
-        throw new SecurityException(String.format("Current request belongs to the organization \"%s\". Expected \"%s\"",
-                org.getId(), organization.getId()));
+        throw new SecurityException(String.format(
+            "Current request belongs to the organization \"%s\". Expected \"%s\"",
+            org.getId(),
```

Comment:
I would join this line and the next.

I would also like to see `org` and `organization` be renamed so they're clearer.  Ugh.  That being said, that particular change is out of scope.